packages/client/src/objectSet/ObjectSet.test.ts

```diff
@@ -295,52 +297,53 @@ describe("ObjectSet", () => {
-
-          expect(result.data).toHaveLength(3);
-          expectTypeOf(result.data[0]).branded.toEqualTypeOf<
-            Osdk<Employee, "$rid" | "$all">
-          >();
-          expectTypeOf(result.data[0]).branded.toEqualTypeOf<
-            Employee.OsdkObject<"$rid">
-          >();
-        });
-      });
-
-      describe("includeRid: false", () => {
-        it("drops bad data", async () => {
-          const opts = {
-            $__EXPERIMENTAL_strictNonNull: "drop",
-            $includeRid: false,
-          } as const;
-          const result = k === "fetchPage"
-            ? await client(Employee).fetchPage(opts)
-            : (await client(Employee).fetchPageWithErrors(opts)).value!;
-
-          expect(result.data).toHaveLength(3);
-          expectTypeOf(result.data[0]).branded.toEqualTypeOf<Osdk<Employee>>();
-          expectTypeOf(result.data[0]).branded.toEqualTypeOf<
-            Employee.OsdkObject
-          >();
-          expectTypeOf(result.data[0]).branded.toEqualTypeOf<
-            Employee.OsdkObject<never>
-          >();
-        });
-      });
-    });
+    // describe("strictNonNull: \"drop\"", () => {
+    //   describe("includeRid: true", () => {
+    //     it("drops bad data", async () => {
+    //       const opts = {
+    //         $__EXPERIMENTAL_strictNonNull: "drop",
+    //         $includeRid: true,
+    //       } as const;
+    //       const result = k === "fetchPage"
+    //         ? await client(Employee).fetchPage(opts)
+    //         : (await client(Employee).fetchPageWithErrors(opts)).value!;
+
+    //       expect(result.data).toHaveLength(3);
+    //       expectTypeOf(result.data[0]).branded.toEqualTypeOf<
+    //         Osdk<Employee, "$rid" | "$all">
+    //       >();
+    //       expectTypeOf(result.data[0]).branded.toEqualTypeOf<
+    //         Employee.OsdkObject<"$rid">
+    //       >();
+    //     });
+    //   });
+
+    //   describe("includeRid: false", () => {
+    //     it("drops bad data", async () => {
+    //       const opts = {
+    //         $__EXPERIMENTAL_strictNonNull: "drop",
+    //         $includeRid: false,
+    //       } as const;
+    //       const result = k === "fetchPage"
+    //         ? await client(Employee).fetchPage(opts)
+    //         : (await client(Employee).fetchPageWithErrors(opts)).value!;
+
+    //       expect(result.data).toHaveLength(3);
+    //       expectTypeOf(result.data[0]).branded.toEqualTypeOf<Osdk<Employee>>();
+    //       expectTypeOf(result.data[0]).branded.toEqualTypeOf<
+    //         Employee.OsdkObject
+    //       >();
+    //       expectTypeOf(result.data[0]).branded.toEqualTypeOf<
+    //         Employee.OsdkObject<never>
+    //       >();
+    //     });
+    //   });
+    // });
 
     describe("strictNonNull: false", () => {
       describe("includeRid: true", () => {
         it("returns bad data", async () => {
+          // look here
```

Comment:
whats this

Reply:
This was comment I added for myself to point out to you later lol, will delete as I address things